api/UserController.js

```diff
@@ -129,6 +130,60 @@ exports.updateUser = function(req, res) {
         })
 };
 
+exports.forgotPassword = function(req, res) {
+    console.log(req.body);
+    return db.User.findOne({
+        where: {email: req.body.email}
+    }).then(user => {
+        if (!user) {
+            return util.detailErrorResponse(res, 404, 'User not found.');
+        } else {
+            emailutil.sendResetPasswordEmail(
+                            user.dataValues.email, 
+                            'info@boardgameweekend.party',
+                            'BoardGameCompanion', 
+                            user.dataValues.name, 
+                            'http://localhost:8080/#/auth/reset_password?token=' + 
+                                userutil.getResetPasswordToken(user.dataValues.id, user.dataValues.email,
```

Comment:
Usually, you don't need to go through dataValues to access model attributes (e.g. user.id, user.email is valid). Some other parts of the code might still contain access through dataValues but it is something I have to change